manager/update_cli_config.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import sys
+import yaml
+import os
+
+
+def update_cli_config(
+    cli_config_file_path,
+    cortex_environment,
+    operator_endpoint,
+    aws_access_key_id,
+    aws_secret_access_key,
+):
+    cli_env_config = {
+        "name": cortex_environment,
+        "operator_endpoint": operator_endpoint,
+        "aws_access_key_id": aws_access_key_id,
+        "aws_secret_access_key": aws_secret_access_key,
+    }
+
+    try:
+        with open(cli_config_file_path, "r") as f:
+            cli_config = yaml.safe_load(f)
+            if cli_config is None:
+                raise
```

Comment:
`raise <empty>` is a syntax error. In this case, it doesn't matter because it behaves correctly, but we should raise a simple exception nonetheless.

Reply:
Haha good catch!